lightly_studio/tests/core/test_add_samples.py

```diff
@@ -199,23 +201,53 @@ def test_create_batch_samples(db_session: Session) -> None:
     assert existing_paths == ["/path/to/image_0.png"]
 
 
-def _get_labelformat_input(filename: str = "image.jpg") -> LabelformatObjectDetectionInput:
+def test_create_label_map(db_session: Session) -> None:
+    # Test the creation of new labels and re-use of existing labels
+    label_input = _get_labelformat_input(filename="image.jpg", category_names=["dog", "cat"])
+
+    label_map = add_samples._create_label_map(
+        session=db_session,
+        input_labels=label_input,
+    )
+
+    label_input_2 = _get_labelformat_input(
+        filename="image.jpg", category_names=["dog", "cat", "bird"]
+    )
+
+    label_map_2 = add_samples._create_label_map(
+        session=db_session,
+        input_labels=label_input_2,
+    )
+
+    assert len(label_map) == 2  # dog and cat
+    assert len(label_map_2) == 3  # dog, cat and bird
+    assert label_map_2[0] == label_map[0]  # dog exists already
+    assert label_map_2[1] == label_map[1]  # cat exists already
+    assert label_map_2[2] != label_map[1]  # bird is new
```

Comment:
This check does not make sense, it tests that bird is not dog. We can e.g. do:
```py
assert label_map_2[2] not in label_map_1.values()
```

Also I suggest to add a comment explaining what we are comparing (annotation_label_id for a category id)